src/services/email.services.js

```diff
@@ -0,0 +1,57 @@
+const createTransport = require('nodemailer').createTransport({
+  service: 'gmail',
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: true,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+const sendActivationLink = async (to, link) => {
+  const transporter = await createTransport.sendMail({
```

Comment:
The `createTransport` function is incorrectly used as `createTransport.sendMail`. Instead, `createTransport` should be called to create a transporter object, and then `sendMail` should be called on that transporter. Correct this by first creating a transporter with `createTransport()` and then using `transporter.sendMail()`.